.github/workflows/publish.yml

```diff
@@ -32,6 +33,10 @@ jobs:
       - run: npm ci
       - run: npm run lint
       - run: npm run build --if-present
+      - run: npm run hardhat:arbitrum & npm run test:arbitrum
+      - run: npm run hardhat & npm run test:utils
+      - run: npm run hardhat & npm run test:builders
+      - run: npm run hardhat & npm run test:quote
```

Comment:
As mentioned wouldn't bother to much with the ci test setup now, if it works fine locally, it's also good to release the SDK later.